### ISSUE TITLE
upgrade to v6.0.3 throws Cannot use import statement outside a module

### ISSUE THREAD

**Issue by vasco3** [magiclabs/magic-js#213]
### ✅ Prerequisites

- [x] Did you perform a cursory search of open issues? Is this bug already reported elsewhere?
- [x] Are you running the latest SDK version?
- [x] Are you reporting to the correct repository (`magic-sdk`)?

### 🐛 Description

[Description of the bug.]

### 🧩 Steps to Reproduce

1. Using next.js import the module magic-sdk into a file.
2. run `npm run build` 

### 🤔 Expected behavior
It should build without errors

### 😮 Actual behavior

throws Cannot use import statement outside a module

```
...nc _post(e){const r=await this.iframe;if(!r||!r.contentWindow)throw t();r.contentWindow.postMessage(e,this.endpoint)}},configureStorage:async()=>{const e=s.createInstance({name:"MagicAuthLocalStorageDB",storeName:"MagicAuthLocalStorage"});return await e.defineDriver(h),await e.setDriver([s.INDEXEDDB,s.LOCALSTORAGE,l._driver]),e}});export{y as Magic};
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at wrapSafe (internal/modules/cjs/loader.js:979:16)
    at Module._compile (internal/modules/cjs/loader.js:1027:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.6791 (/Users/jorgec/Downloads/code/sobras/.next/server/pages/inicio.js:52:44)
    at __webpack_require__ (/Users/jorgec/Downloads/code/sobras/.next/server/webpack-runtime.js:25:42)
    at /Users/jorgec/Downloads/code/sobras/.next/server/pages/inicio.js:17:80 {
  type: 'SyntaxError'
```
### 💻 Code Sample

[If possible, please provide a code repository, gist, code snippet or sample files to reproduce the issue.]

### 🌎 Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| `magic-sdk`      | 6.0.3
| Browser          | any
| `yarn`           |  npm
| Operating System | MacOS 11.5.2
| nodjs | v14.16.1

**Comment by smithki** [comment]
:rocket: Issue was released in [`@magic-sdk/commons@2.0.4`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-sdk/commons%402.0.4), [`@magic-sdk/provider@6.0.4`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-sdk/provider%406.0.4), [`@magic-sdk/react-native@6.0.4`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-sdk/react-native%406.0.4), [`@magic-sdk/types@5.0.2`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-sdk/types%405.0.2), [`magic-sdk@6.0.4`](https://github.com/magiclabs/magic-js/releases/tag/magic-sdk%406.0.4) :rocket:

**Comment by smithki** [comment]
:rocket: Issue was released in [`@magic-sdk/commons@3.0.0-dgerrells-ch42053-sms-feature.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-sdk/commons%403.0.0-dgerrells-ch42053-sms-feature.0), [`@magic-sdk/provider@7.0.0-dgerrells-ch42053-sms-feature.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-sdk/provider%407.0.0-dgerrells-ch42053-sms-feature.0), [`@magic-sdk/react-native@7.0.0-dgerrells-ch42053-sms-feature.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-sdk/react-native%407.0.0-dgerrells-ch42053-sms-feature.0), [`@magic-sdk/types@6.0.0-dgerrells-ch42053-sms-feature.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-sdk/types%406.0.0-dgerrells-ch42053-sms-feature.0), [`magic-sdk@7.0.0-dgerrells-ch42053-sms-feature.0`](https://github.com/magiclabs/magic-js/releases/tag/magic-sdk%407.0.0-dgerrells-ch42053-sms-feature.0) :rocket:
